### DISCUSSION TITLE
I cant export gltf on a single file WITH the texture inside

### DISCUSSION THREAD

**Discussion by LeandroKanishka** [iimachines/Maya2glTF#190] [Q&A]
Hi! thank for the plugin !!!  i cant export all (my mesh and the textures selected in the GLSL Shader material) on One big file... the export button giveme my gltf file and the textures separatedly.   Can i export one single file wich include the textures?   Thanks!!

**Comment by ziriax** [comment]
Did you try entering `-glb` in the extra flags text input box of the exporter UI?
